config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -1342,6 +1342,21 @@ public HttpSecurity authorizeHttpRequests(
 		return HttpSecurity.this;
 	}
 
+	/**
+	 * get authorizeHttpRequests with default values
+	 * @return the {@link HttpSecurity} for further customizations
+	 * @throws Exception
+	 * @see #authorizeHttpRequests(Customizer)
```

Comment:
Please also add `@since 5.5` since that is the version where it will be added.

Reply:
Done